benchmark.py

```diff
@@ -40,16 +43,38 @@
 
 checkpoint = torch.load(args.checkpoint, map_location = 'cpu') if args.checkpoint else None
 if checkpoint:
-	args.model, args.lang, args.sample_rate, args.window_size, args.window_stride, args.window, args.num_input_features = map(checkpoint['args'].get, ['model', 'lang', 'sample_rate', 'window_size', 'window_stride', 'window', 'num_input_features'])
+	args.model, args.lang, args.sample_rate, args.window_size, args.window_stride, args.window, args.num_input_features = map(
+		checkpoint['args'].get,
+		['model', 'lang', 'sample_rate', 'window_size', 'window_stride', 'window', 'num_input_features'])
 
 use_cuda = 'cuda' in args.device
 
 labels = datasets.Labels(datasets.Language(args.lang))
 
+if args.profile_cuda:
```

Comment:
this would affect the profiled timings as well. please introduce a new argparse flag

Reply:
done